src/components/Dashboard.vue

```diff
@@ -217,9 +217,14 @@
         const lastUpdate = this.syncData.last_extended > this.syncData.last_soc ?
           this.syncData.last_extended : this.syncData.last_soc;
 
-        this.dataOutdatedMessage =
-          `Data outdated. Updated ${this.$root.MomentJS(new Date(lastUpdate * 1000)).fromNow()}.`;
-        this.dataOutdatedMessageTimestamp = `(${this.$root.MomentJS(new Date(lastUpdate * 1000)).format('MMMM Do YYYY HH:mm')})`;
+        if(!lastUpdate) {
+          this.dataOutdatedMessage = `There has never been a connection to a car. Please connect your car first time.`
+        }
+        else {
```

Comment:
Just a code style guideline (sorry, currently not defined officially :D ):
can you move the `else` after the bracket?

so:
if (..) {

} else {

}

Otherwise it looks fine :)

Reply:
Ok changed.